tracing/README.md

```diff
@@ -0,0 +1,188 @@
+Distributed Tracing
+---
+
+Both Helidon SE and Helidon MP use Opentracing API for tracing
+events.
+
+_Eclipse Microprofile Opentracing specification is not yet implemented_
+
+#Usage
+## Usage In Helidon MP
+To use tracing in MP, simply depend on the `helidon-microprofile-tracing` module to automate
+setup of tracing, and add your favorite tracer implementation to the classpath (currently Zipkin is the only
+favorite tracer implemented, though SPI is available).
+
+Example of pom.xml dependencies:
+```xml
+<dependency>
+    <!-- general support for tracing -->
+    <groupId>io.helidon.microprofile.tracing</groupId>
+    <artifactId>helidon-microprofile-tracing</artifactId>
+</dependency>
+<dependency>
+    <!-- Zipkin tracer implementation -->
+    <groupId>io.helidon.tracing</groupId>
+    <artifactId>helidon-tracing-zipkin</artifactId>
+</dependency>
+```
+
+To customize configuration of zipkin, see `ZipkinTracerBuilder` javadoc. Basics:
+```yaml
+tracing:
+  # required
+  service: "service-name"
+  # default is "localhost"
+  host: "zipkin"
+  # default is 9411
+  port: 9411
+```
+
+## Usage in Helidon SE
+To use tracing in SE, add a dependency on `helidon-tracing` and a tracer
+implementation and register tracer with server configuration.
+
+pom.xml:
+```xml
+<dependency>
+    <!-- to add tracer builder -->
+    <groupId>io.helidon.tracing</groupId>
+    <artifactId>helidon-tracing</artifactId>
+</dependency>
+<dependency>
+    <!-- to add zipkin support -->
+    <groupId>io.helidon.tracing</groupId>
+    <artifactId>helidon-tracing-zipkin</artifactId>
+</dependency>
+```
+
+code using config:
+```java
+return ServerConfiguration.builder()
+                .config(config.get("webserver"))
+                .tracer(TracerBuilder.create(config.get("tracing"))
+                                        .buildAndRegister())
+                .build();
+```
+
+To customize configuration of zipkin, see `ZipkinTracerBuilder` javadoc. Basics:
```

Comment:
"To customize the configuration of Zipkin, see ..."

And move "Basics:" to a separate line.